src/figure/FigureStateActive.js

```diff
@@ -125,7 +127,18 @@ const initForm = ({ widget, formElement, formTpl, mediaEditor, togglePlaceholder
     );
 };
 
-export default function (stateFactory, ActiveState, formTpl, formElement, inlineHelper) {
+/**
+ * 
+ * @param {Object} config
+ * @param {Object} config.stateFactory
+ * @param {Object} config.ActiveState
+ * @param {Object} config.formTpl
+ * @param {Object} config.formElement 
+ * @param {Object} config.inlineHelper
+ * @param {Object} config.htmlEditor
+ * @returns 
+ */
+export default function (config) {
```

Comment:
**suggestion:** It could also have been turned into a restructured declaration as mentioned in [the coding guide](https://github.com/oat-sa/taohub-articles/blob/master/articles/Frontend/Architecture/coding-guide.md#es2015-style)
```suggestion
export default function ({ stateFactory, ActiveState, formTpl, formElement, inlineHelper, htmlEditor }) {
```

If some parameters are mandatory and need to be given anyway, it could also be split into groups:
```suggestion
export default function (stateFactory, ActiveState, { formTpl, formElement, inlineHelper, htmlEditor }) {
```